ui/webui/test/prepare-updates-img

```diff
@@ -2,18 +2,37 @@
 
 set -eu
 
+# FIXME boot.iso on rawhide does not currently contain the new cockpit dependencies
+# This will change once we include this changes upstream and start building boot.iso with the new dependencies
+# Then we can safely remove this workaround
+ANACONDA_WEBUI_DEPS_URLS='
+    https://kojipkgs.fedoraproject.org//packages/cockpit/292/1.fc39/x86_64/cockpit-ws-292-1.fc39.x86_64.rpm
```

Comment:
So the reason for no longer getting these RPMs from direct links was that the links eventually go dead and everything breaks. Are we sure this change is temporary enough so this will not be an issue or perhaps some other solution will be better ?

Maybe we can download the PRMs to a folder on fedorapeople and fetch from there ?